src/utils/update/review.beneficial.owner.ts

```diff
@@ -0,0 +1,96 @@
+import { BeneficialOwnerGov } from "../../model/beneficial.owner.gov.model";
+import * as config from "../../config";
+import { ApplicationData } from "../../model";
+import { BeneficialOwnerIndividual } from "../../model/beneficial.owner.individual.model";
+
+const AllBoTypes = {
+  boiReview: "review_beneficial_owners_individual",
+  boGovReview: "review_beneficial_owners_government_or_public_authority",
+  boIndividual: "beneficial_owners_individual",
+  boGov: "beneficial_owners_government_or_public_authority"
+};
+
+const beneficialOwnerReviewRedirectUrl = `${config.UPDATE_AN_OVERSEAS_ENTITY_URL
```

Comment:
can we mention 'Individual' or 'Ind' or something in this variable name

Reply:
Fixed